container/docker/handler.go

```diff
@@ -242,17 +273,22 @@ func (self *dockerContainerHandler) getFsStats(stats *info.ContainerStats) error
 	if self.ignoreMetrics.Has(container.DiskUsageMetrics) {
 		return nil
 	}
+	var device string
 	switch self.storageDriver {
+	case devicemapperStorageDriver:
```

Comment:
Why are you adding the fs logic here, instead of abstracting all of it in `FsHandler`?